.github/workflows/conda-python-build.yaml

```diff
@@ -13,6 +13,11 @@ on:
         type: string
         description: "relative path to a script that builds conda packages"
 
+# override default permissions
+permissions:
+  # needed to auth with AWS for sccache
+  id-token: read
```

Comment:
Attempting to fix this:

> It looks like you might be trying to authenticate with OIDC. Did you mean to set the `id-token` permission? If you are not trying to authenticate with OIDC and the action is working successfully, you can ignore this message.

https://github.com/rapidsai/legate-raft/actions/runs/13526520778/job/37798438395?pr=15#step:4:13

Reply:
`read` is not an accepted value for `permissions.id-token:`, I guess.

> The workflow is not valid. In .github/workflows/pr.yaml (Line: 40, Col: 11): Error from called workflow rapidsai/legate-raft/.github/workflows/conda-python-build.yaml@af527d6a435c52acf862044ffb78571a393a796c (Line: 19, Col: 13): Unexpected value 'read'

ref: https://github.com/rapidsai/legate-raft/actions/runs/13526669944

So pushed https://github.com/rapidsai/legate-raft/pull/15/commits/ccd14bf7df20b817703cbf865fa84d37c97942e4 trying `write`.